doc/usecases/membership_application.md

```diff
@@ -0,0 +1,23 @@
+# Use Case: Membership Application
+
+## Summary
+
+- **Scope:** Membership
+- **Level:** User Goal
+- **Actors:** App User
+- **Brief:** The user can apply for a membership (account)
```

Comment:
When we talk about the term 'membership', do we simply mean an account or an additional form of club membership?
@markus2330 I think what we want to incentivize with some of the gamification use cases differs from the definition of a membership here or am I mistaken?

Reply:
> When we talk about the term 'membership', do we simply mean an account or an additional form of club membership?

Account implies the membership of the "Verein", so it is correctly written here. Membership of the "Verein", however, does not imply that the person actually pays something. There are also ways to earn the membership without payment (and instead do e.g. actions as described in the gamification membership use case).

> @markus2330 I think what we want to incentivize with some of the gamification use cases differs from the definition of a membership here or am I mistaken?

Nearly all gamification use cases are to improve motivation/behavior within the app/the community. Only a single gamification use case is about earning the free membership.

Reply:
But if I understand correctly, you need a membership to login and use the app. And you get a free membership through using the app. Does that mean a normal user has to pay before he is able to get a free membership?

Reply:
No, if you apply for a free membership, you don't have to pay in the beginning. Only if you don't collect enough points within the year, you either need to pay or your membership will expire.

Reply:
Aah, so like a free trial with the option to extend the duration through using the app, I understand.